recipes/palantir-sdk/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "palantir-sdk" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6652b20d0e35f53942c63a2dc5699603cc09f84eef20691d628cc22bdb0efcb1
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

These flags are handled automatically by environment variables these days:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vvv"
```